testing/run_all_unittests.cc

```diff
@@ -49,6 +49,7 @@ int main(int argc, char** argv) {
 #endif  // FML_OS_IOS
 
   ::testing::InitGoogleTest(&argc, argv);
+  GTEST_FLAG_SET(death_test_style, "threadsafe");
```

Comment:
Had to add this flag to allow `EXPECT_EXIT`.
https://github.com/google/googletest/blob/main/docs/advanced.md#death-test-styles

Otherwise:
```
[WARNING] ../../third_party/googletest/googletest/src/gtest-death-test.cc:1102:: Death tests use fork(), which is unsafe particularly in a threaded context. For this test, Google Test detected 2 threads. See https://github.com/google/googletest/blob/main/docs/advanced.md#death-tests-and-threads for more explanation and suggested solutions, especially if this is the last message you see before your test times out.
../../flutter/shell/platform/darwin/common/framework/Source/flutter_codecs_unittest.mm:52: Failure
Death test: [codec encode:value]
    Result: died but not with expected error.
  Expected: contains regular expression "failed to convert to UTF8"
Actual msg:
[  DEATH   ] 2024-01-29 18:38:36.256 framework_common_unittests[31544:1799550] *** Assertion failure in -[FlutterJSONMessageCodec encode:], FlutterCodecs.mm:82
[  DEATH   ] objc[31544]: +[NSNumber initialize] may have been in progress in another thread when fork() was called.
[  DEATH   ] objc[31544]: +[NSNumber initialize] may have been in progress in another thread when fork() was called. We cannot safely call it or ignore it in the fork() child process. Crashing instead. Set a breakpoint on objc_initializeAfterForkError to debug.
```